File: docs/user/en/query/filtering/examples/string/attribute-ends-with.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					attributeEndsWith("code", "solar")
				),
				require(
					entityFetch(
						attributeContent("code")
					)
				)
			)
		);
	}
);

File: documentation/user/en/query/filtering/examples/string/attribute-ends-with.cs
EvitaResponse<ISealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		AttributeEndsWith("code", "solar")
        	),
        	Require(
        		EntityFetch(
        			AttributeContent("code")
        		)
        	)
        )
	)
);